resources/cfssl-new-cert-etcd.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+set -o errexit
+
+mkdir -p ${path}
+cd ${path}
+
+_ip="$(${get_ip})"
+_hostname="$(hostname)"
+
+# workaround for https://github.com/kubernetes/kubernetes/issues/72102
+# include first member's ip in SAN for all nodes
+# this replicates kubeadm behaviour to include first node's ip, as kubeadm
+# generates all certificates on the first node
+_first_member_ip="$(echo $${_ip} | sed 's/[0-9]*$/4/')"
```

Comment:
too brave, especially when we have `etcd_addresses` as an input list variable.

Reply:
Fixed, using etcd_addresses list